TravelProject/TravelProject/Info.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TravelProject
+{
+    public class Info : DataBase
+    {
+
+        public string _ofiiceLocation;
```

Comment:
Public fields better to convert to properties and reduce the visibility of set accessor.